packages/x-components/src/components/modals/base-id-modal-open.vue

```diff
@@ -1,36 +1,39 @@
 <template>
-  <BaseEventButton
-    v-on="$listeners"
-    :events="events"
-    class="x-button x-events-modal-id-open-button"
-    data-test="open-modal-id"
-  >
-    <!-- @slot (Required) Button content with a text, an icon or both -->
-    <slot />
-  </BaseEventButton>
+  <NoElement v-on="$listeners" data-test="open-modal-id">
+    <!--
+      @slot opening-element. It's the element that will trigger the modal opening. It's a
+      button by default.
+        @binding {Function} openModal - The function to open the modal.
+    -->
+    <slot :openModal="emitOpenModalEvent" name="opening-element">
+      <button @click="emitOpenModalEvent" class="x-button x-events-modal-id-open-button">
+        <slot />
```

Comment:
Add slot documentation as you did un the `BaseIdModalClose` component

```suggestion
        <!-- @slot (Required) Button content with a text, an icon or both -->
        <slot />
```

Reply:
🤦 How did I mess this up?

Reply:
This is still not done